Python/sysmodule.c

```diff
@@ -3479,6 +3479,66 @@ make_impl_info(PyObject *version_info)
     return NULL;
 }
 
+static PyObject*
+make_abi_features(void)
+{
+    int res;
+    PyObject *features = PyList_New(0);
+    if (features == NULL) {
+        goto error;
+    }
+
+#if defined(Py_GIL_DISABLED)
+    PyObject *threading_feature = PyUnicode_FromString("free-threading");
+#else
+    PyObject *threading_feature = PyUnicode_FromString("gil-enabled");
+#endif
+    if (threading_feature == NULL) {
+        goto error;
+    }
+    res = PyList_Append(features, threading_feature);
+    Py_DECREF(threading_feature);
+    if (res < 0) {
+        goto error;
+    }
+
+#if defined(Py_DEBUG) || (defined(MS_WINDOWS) && defined(_DEBUG))
```

Comment:
I'm not 100% sure about this, but I think `#ifdef Py_DEBUG` should suffice here.